components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -152,6 +152,7 @@ public abstract class FrameworkConstants {
     public static final String ORGANIZATION_USER_PROPERTIES = "ORGANIZATION_USER_PROPERTIES";
     public static final String ORGANIZATION_AUTHENTICATOR = "OrganizationAuthenticator";
     public static final String ORGANIZATION_LOGIN_HOME_REALM_IDENTIFIER = "OrganizationSSO";
+    public static final String ORGANIZATION_LOGIN_IDP_NAME = "SSO";
```

Comment:
Why this is required?

Reply:
There would be chances to change the IDP name

Reply:
The IDP name has to be treated as system reserved IDP name as raised in this issue https://github.com/wso2/product-is/issues/20445